EnglishTranslation-master/ETC.tsv

```diff
@@ -17491,10 +17491,10 @@ ETC_20151224_017492	Bomb Score
 ETC_20151224_017493	It has a same meaning as the name. There is no special thing.
 ETC_20151224_017494	They are surely Vubbes, but they live another life.
 ETC_20151224_017495	It is hard to determine whether their mining roles are permanent since little is known about the Vubbe's social system.
-ETC_20151224_017496	At first time, It was created by magical mistakes and abusing, after that it has been intentionally created by the malicious wizard or demon.
+ETC_20151224_017496	At first, it was created by magical mistakes and abuse. After that it has been intentionally created by the malicious wizard or demon.
```

Comment:
how about changing it to this (I suppose this is a monster description): 
`At first, they were created accidentally through the abuse of magic, but later malicious wizards and demons started creating them intentionally.`